### PR TITLE
Small Updates to Sidenav

### PR DESCRIPTION
**Pull request by plhnk**
starting a PR for minor visual tweaks as I familiarize myself w/repo 
- improve legibility on nav items --> style icons separately from text
- add transitions on hovers for nav items
- align border for nested items to horizontal center of icon

### CODE REVIEW

File: src/components/sidenav/sidenav.tsx [hunk 1]
```diff
@@ -28,9 +28,9 @@ import Link, { LinkProps } from "next/link";
 /**
  * Shared styles for both list items and nested list items
  */
-const _liBase = `inline-flex flex-row items-center flex-1 bg-transparent rounded`;
-const _liTypography = `text-sm/4 font-semibold text-white`; // #TODO: non-standard text style
-const _liHover = `cursor-pointer hover:bg-blue-500`;
+const _liBase = `group inline-flex flex-row items-center flex-1 bg-transparent rounded transition-colors duration-300`;
+const _liTypography = `text-sm/4 font-semibold text-blue-50`; // #TODO: non-standard text style
+const _liHover = `cursor-pointer hover:bg-blue-500 hover:text-white transition-colors duration-300`;
```
Comment (rttnbrgr) [inline src/components/sidenav/sidenav.tsx:33]: These changes ripple out to the client portal nav as well.

Take a look at how it's affected those styles and let's see how to best address that.

File: src/components/sidenav/sidenav.tsx [hunk 2]
```diff
@@ -110,7 +110,17 @@ export const SideNavItem = ({
...
+        <span
+          className={
+            isExpanded
+              ? "text-blue-200 group-hover:text-blue-100"
+              : "text-blue-100 group-hover:text-blue-50"
+          }
+        >
+          {icon}
+        </span>
+      )}
```
Comment (rttnbrgr) [inline src/components/sidenav/sidenav.tsx:123]: Is this adding a slight color variation between the icon and the nav text? @plhnk

File: src/components/sidenav/sidenav.tsx [hunk 3]
```diff
@@ -428,7 +438,7 @@ export const NavHeader = ({
       </div>
       <button
         onClick={() => {
-          setExpanded(cv => !cv);
+          setExpanded((cv) => !cv);
```
Comment (rttnbrgr) [inline src/components/sidenav/sidenav.tsx:441]: this seems like its prettier differences? May want to pull in the latest from main as we now have a `.prettierrc` in the repo

File: src/components/sidenav/sidenav.tsx [hunk 4]
```diff
@@ -500,7 +510,7 @@ export const SideNav = ({ ...props }) => {
                 isNested={item.type === "nested"}
                 isActive={open === item.title}
                 isExpanded={expanded}
-                onClick={e => {
+                onClick={(e) => {
```
Comment (rttnbrgr) [inline src/components/sidenav/sidenav.tsx:513]: ditto the above

**Review by rttnbrgr — CHANGES_REQUESTED** [review]
Added my notes.

### Couple follow up things:

- Would love to get any changes we make, into figma as well. If iterating in code is faster for you, I'm good with that, let's just make sure changes get unified.  
- Feel free to use the main repo. I'm not sure the exact benefit or drawbacks of using a fork, but please do not feel restricted from hopping into the main repo. 

Awesome to see you digging into to all this @plhnk

**Comment by plhnk** [comment]
closing out as hardcoding color values clashes with the other sidebars - will pick any refinements after semantic color layer is in place

**Pull request closed without merging**